README.md

```diff
@@ -20,9 +20,35 @@ BRANCH_NAME=master
 curl --location --remote-name https://github.com/sgmap/mes-aides-ops/archive/$BRANCH_NAME.tar.gz
 tar -xvf $BRANCH_NAME.tar.gz
 cd mes-aides-ops-$BRANCH_NAME
-./bootstrap.sh
+# ./bootstrap.sh mes-aides-ui_head mes-aides-ops_head
+./bootstrap.sh origin/master origin/$BRANCH_NAME
 ```
 
+`./bootstrap.sh origin/master origin/$BRANCH_NAME` initiates the set-up. By default, `origin/master` is used for both repositories (https://github.com/sgmap/mes-aides-ui and https://github.com/sgmap/mes-aides-ops). That can be overidden by passing *TREEISH* parameters. The fist one is `mes-aides-ui` head and the second one is `mes-aides-ops` one.
+
+That is why the suggested set of commands above overrides the default head of `mes-aides-ops` to rely on the selected branch.
+
+## Continuous provisioning and deployment
+
+### Provisioning
+
+A private key can `ssh` to the host and it will automatically trigger:
```

Comment:
This is ambiguous: “a private key”? A specific one or any private key that can connect to the host?